boards/arc/arduino_101_sss/arduino_101_sss.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: arduino_101_sss
+name: arduino_101_sss
```

Comment:
its not the same, I did most of the boards, not all of them though. I do not want to rely on file names, we have a structured syntax, why not use that, otherwise it will be confusing and we will be maintaining information in different ways.